src/content/components/Tooltip.css

```diff
@@ -0,0 +1,69 @@
+.tooltipMount {
+  /* Create a stacking context above the root container element. */
+  position: absolute;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  pointer-events: none;
+}
+
+.tooltip {
+  position: relative;
+  max-width: 100%;
+  padding: 2px 5px;
+  display: inline-block;
+  border: 1px solid #ccc;
+  background-color: #f9f9f9;
+  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.3);
+  text-align: left;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  transition: opacity 250ms;
+  box-sizing: border-box;
+}
+
+.tooltipTiming {
+  padding-right: 0.5em;
+  color: var(--theme-highlight-green);
+  font-weight: bold;
+}
+
+.tooltipOneLine {
+  display: flex;
+  max-width: 100%;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+}
+
+.tooltipName {
+  flex: 1;
+  text-overflow: ellipsis;
+  overflow: hidden;
+}
+
+.tooltipSwatch {
+  width: 10px;
+  height: 10px;
+  display: inline-block;
+  border: 1px solid #888;
+  margin-right: 3px;
+}
+
+.tooltipHeader {
+  padding-bottom: 10px;
+  margin-bottom: 10px;
+  border-bottom: 1px solid var(--theme-highlight-gray);
+}
+
+.tooltipDetails {
+  margin: 5px 0;
+}
+
+.tooltipLabel {
+  display: block;
+  width: 60px;
+  color: var(--theme-highlight-gray);
```

Comment:
this is way too pale for accessibility. See https://snook.ca/technical/colour_contrast/colour.html#fg=B4BABF,bg=FFFFFF